resources/views/admin/blocks/showcase.blade.php

```diff
@@ -1,63 +1,283 @@
+@php
+    $currentUrl = explode('/', request()->url());
+    $type = \App\Models\LandingPage::find(intval($currentUrl[4]))->type;
```

Comment:
This seems like an unreliable way to determine the LandingPage model the block is working with. Is there a `$block` variable available to you here you can inspect to get the model you're working with?